Sources/NIOFileSystem/FileInfo.swift

```diff
@@ -145,6 +147,29 @@ extension FileInfo {
 
     /// A time interval consisting of whole seconds and nanoseconds.
     public struct Timespec: Hashable, Sendable {
+        #if canImport(Darwin)
+        private static let UTIME_OMIT_INT = Int(UTIME_OMIT)
+        private static let UTIME_NOW_INT = Int(UTIME_NOW)
+        #elseif canImport(Glibc) || canImport(Musl)
+        private static let UTIME_OMIT_INT = Int(CNIOLinux_UTIME_OMIT)
+        private static let UTIME_NOW_INT = Int(CNIOLinux_UTIME_NOW)
+        #endif
+
+        /// A timespec where the seconds are set to zero and the nanoseconds set to `UTIME_OMIT`.
+        /// In syscalls such as `futimens`, this means the time component set to this value will be ignored.
+        public static var omit = Self.init(
```

Comment:
🚨 `public static var` alert! 🚨

This should either be a `let` or a computed property. Same goes for `now`.

Reply:
Also let's avoid `Self.init(...)` and just use `Self(...)`

Reply:
Easily done 🙂